content/en/docs/concepts/configuration/secret.md

```diff
@@ -348,7 +348,7 @@ key authentication:
 
 The SSH authentication Secret type is provided only for convenience.
 You can create an `Opaque` type for credentials used for SSH authentication.
-However, using the defined and public Secret type (`kubernetes.io/ssh-auth`) helps other
+However, using the defined and public Secret type (`kubernetes.io/tls`) helps other
```

Comment:
@Arhell Could you provide additional details regarding this change? It appears that the section pertains to ['SSH authentication Secrets'](https://kubernetes.io/docs/concepts/configuration/secret/#ssh-authentication-secrets), and the `kubernetes.io/ssh-auth` Secret type seems appropriate without requiring any alterations.